tests/devapps/NetCoreTestApp/Program.cs

```diff
@@ -58,7 +58,10 @@ public static void Main(string[] args)
             Logger.LogCallback = Log;
             Logger.Level = LogLevel.Verbose;
             Logger.PiiLoggingEnabled = true;
-
+#if TELEMETRY
+            Telemetry.GetInstance().RegisterReceiver(
+                new Microsoft.Identity.Client.DevAppsTelemetry.ServerTelemetryHandler().OnEvents);
```

Comment:
This is a bit difficult to read - I assume that you are invoking the constructor of ServerTelemetryHandler and then passing in the OnEvents delegate that lives on the new instance to RegisterReceiver? Maybe if you add some paranthesis around the ctor invocation it would help.,